dashboard/app/models/level_group.rb

```diff
@@ -87,4 +88,135 @@ def pages
   def plc_evaluation?
     levels.map(&:class).uniq == [EvaluationMulti]
   end
+
+  # Surveys: How many students must complete a survey before any results are shown.
+  SURVEY_REQUIRED_SUBMISSION_COUNT = 5
+
+  # Surveys: Given a sublevel, and the known response to it, return a result object.
```

Comment:
Interesting.  I prepped this change just now, but it's hard to imagine anything but a LevelGroup having this specialized survey support, and I'd rather keep all the survey functions together for now, at least for as long as they are the only ones that call each other.  Still could be convinced otherwise.